api_specifications/case-management-api-0.1.0.yaml

```
@@ -283,4 +337,79 @@ components:
+        startTime:
+          type: integer
+          format: int64
+          description: The date time of start of case bundle creation in epoch time (milliseconds)
+        endTime:
+          type: integer
+          format: int64
+          description: The date time of end of case bundle creation in epoch time (milliseconds)
+        caseCount:
+          type: integer
+          format: int32
+          description: the count of the cases retrieved from the index, whose bundle needs to be created
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)